ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.test.tsx

```diff
@@ -115,7 +127,92 @@ describe('TimeSeriesChartPanel', () => {
 
   it('should toggle selected state when a legend item is clicked', async () => {
     renderPanel();
-    await userEvent.click(screen.getByTestId('legend-item-hsla(-141599372,50%,50%,0.8)'));
-    expect(screen.getByTestId('legend-item-hsla(-141599372,50%,50%,0.8)')).toHaveClass('Mui-selected');
+
+    const seriesArr = Array.from(MOCK_TIME_SERIES_DATA.series);
```

Comment:
Need to do this to make typescript happy because the series is typed as an iterable, not an array.

Reply:
Yeahh we should maybe just change that `Iterable<GraphSeriesValueTuple>` type at some point